gobcore/events/import_events.py

```diff
@@ -0,0 +1,196 @@
+"""GOB Events
+
+Each possible event in GOB is defined in this module.
+The definition includes:
+    name - the name of the event, e.g. ADD
+    timestamp - the name of the timestamp attribute in the GOB entity that tells when this event has last been applied
+    get_modification - method to
+
+todo: The delete and confirm actions contain too much data. Contents can be left empty
+    A deletion or confirmation simple specifies source and sourceid in a collection
+    See also the examples in the action classes
+
+"""
+from abc import ABCMeta, abstractmethod
+
+from gobcore.exceptions import GOBException
+
+modifications_key = 'modifications'
+
+
+class ImportEvent(metaclass=ABCMeta):
+    name = "event"
+    is_add_new = False
+    timestamp_field = None  # Each event is timestamped
+
+    @classmethod
+    @abstractmethod
+    def create_event(cls, _source_id, _id_column, _entity_id, data):
+        """Creates the event dict for the given parameters
```

Comment:
The same here: an event normally applies to an entity identified by it's source and it's source_id. But considering the todo statement at the begin of this module the code can be left as is

Reply:
Will leave a todo